lib/wallet/rpc.js

```diff
@@ -1232,7 +1232,7 @@ class RPC extends RPCBase {
     const valid = new Validator(args);
     let name = valid.str(0);
     const count = valid.u32(1, 10);
-    const from = valid.u32(2, 0);
+    const skip = valid.u32(2, 0);
```

Comment:
We should also change the argument name in the help message up on line 1228

Reply:
Good call, adding this in